.eslintrc.js

```diff
@@ -3,8 +3,32 @@ module.exports = {
     overrides: [
         {
             files: ['*.ts', '*.tsx'],
+            plugins: ['simple-import-sort'],
             rules: {
                 'react/prop-types': 'off',
+                'simple-import-sort/imports': [
```

Comment:
I think we should add this to `@deriv/eslint-config-deriv` npm package so that these rules can be followed on our other repos too.
https://github.com/binary-com/eslint-config-deriv/blob/main/index.js

Reply:
@balakrishna-deriv , yes good idea, will create a PR against that and close this PR 👍